lib/context.js

```diff
@@ -18,6 +18,7 @@ function Context (schema, handler, Reply, Request, contentTypeParser, config, er
   this.onResponse = null
   this.config = config
   this.errorHandler = errorHandler || defaultErrorHandler
+  this.defaultErrorHandler = defaultErrorHandler
```

Comment:
There is no need to allocate this property. `defaultErrorHandler` is a static function, you can require it inside `reply`.